src/App.tsx

```diff
@@ -1,61 +1,150 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todoList, setTodo] = useState(todosFromServer);
+  const [todoState, setData] = useState({
+    title: '',
+    select: '0',
+  });
+  const [titleError, setTitileError] = useState(false);
+  const [selectError, setSelectError] = useState(false);
+
+  const handleChange = (event:
+  React.ChangeEvent<HTMLSelectElement> |
+  React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleChange = (
    event: React.ChangeEvent<HTMLSelectElement> | React.ChangeEvent<HTMLInputElement>
  ) => {
```
You can also try `React.ChangeEvent<HTMLSelectElement | HTMLInputElement>`